docs/source/form-config.rst

```diff
@@ -0,0 +1,78 @@
+Form configuration files
+========================
```

Comment:
```suggestion
Form configuration
==================
```

For me the term “file“ seems a bit to narrow. Eg. an API-response wouldn’t be a file in my understanding.